playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -3,6 +3,20 @@ env_id: "openshift"
 public_dns_domain: "example.com"
 public_dns_nameservers: []
 
+# FIXME(bogdando) transitioning variable names for compatibility with
+# https://github.com/openshift/openshift-ansible/pull/6039 namespaces
+# remove it once resubmitted by the new location in openshift-ansible
+openshift_openstack_clusterid: "{{ env_id }}"
+openshift_openstack_public_dns_domain: "{{ public_dns_domain }}"
+
+# Allows to override the suffix provided in openshift_openstack_clusterid
+# and the private DNS domain.
+openshift_openstack_public_hostname_suffix: "{{ openshift_openstack_clusterid }}"
+openshift_openstack_private_hostname_suffix: "{{ openshift_openstack_clusterid }}"
+openshift_openstack_private_dns_domain: "{{ openshift_openstack_public_dns_domain }}"
```

Comment:
Can we set a default value so this doesn't break existing users?

Reply:
it has it at the line 3